bucket/libreoffice.json

```diff
@@ -1,62 +1,69 @@
 {
-    "version": "7.3.2",
-    "description": "Powerful office suite",
+    "version": "7.3.4",
+    "description": "Powerful and free office suite, a successor to OpenOffice(.org).",
     "homepage": "https://libreoffice.org/",
     "license": "MPL-2.0",
     "suggest": {
-        "vcredist": "extras/vcredist2022"
+        "Visual C++ Redistributable 2015-2022": "extras/vcredist2022"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.documentfoundation.org/libreoffice/stable/7.3.4/win/x86_64/LibreOffice_7.3.4_Win_x64.msi",
+            "hash": "509c70c1c8136805480146b55e4bad5dc73b11ee47b4682b43cf07670109e176"
+        },
+        "32bit": {
+            "url": "https://download.documentfoundation.org/libreoffice/stable/7.3.4/win/x86/LibreOffice_7.3.4_Win_x86.msi",
+            "hash": "e73e2fd3d95f305a8517abe65c940e2b7de13d4271fb789f38f89c28df57933a"
+        }
     },
-    "url": "https://download.documentfoundation.org/libreoffice/portable/7.3.2/LibreOfficePortable_7.3.2_MultilingualStandard.paf.exe#/dl.7z",
-    "hash": "ae81da2a2bfb27a0cbdeffa65f712f8483a4f20049cbd1e02de2a36e6c3ff9d5",
     "pre_install": [
```

Comment:
Is there anything we're missing from the pre_install of `libreoffice-fresh` (the old package that used the .msi installer too)? Or is that all covered already?
https://github.com/ScoopInstaller/Extras/blob/613239a4913ea2b16a7f4fa9eac2c480968759bf/deprecated/libreoffice-fresh.json#L19-L27

Reply:
> Is there anything we're missing from the pre_install of `libreoffice-fresh` (the old package that used the .msi installer too)? Or is that all covered already?

Thanks for the details.

I think we should add this.
I tried to extract the file with `lessmsi`, and the app **is** located under `LibreOffice` folder.
This can avoid problems when user sets **MSIEXTRACT_USE_LESSMSI** to **true**. (Although it should be a rare case)

Reply:
If we add this and only extract the `LibreOffice` folder will we not get the fonts in the `Fonts` folder?